bin/ttl-to-json

```diff
@@ -103,8 +105,7 @@ fileAccessible(cmds.ttlfile);
 if (!_.isUndefined(cmds.frame)) {
    if (_.isNull(cmds.frame)) {
        console.error("Frame argument requires a frame file!");
-       help();
-       process.exit(1);
+       help(1);
```

Comment:
if help always exits, maybe change to "helpAndExit(1)" or use a variable exitStatus (or a comment) so it's clearer what you're passing 1 for.